stdlib/public/core/UnsafeRawBufferPointer.swift.gyb

```diff
@@ -1207,17 +1207,47 @@ extension ${Self} {
 ///     of the closure's execution.
 /// - Returns: The return value, if any, of the `body` closure.
 @inlinable
-public func withUnsafeMutableBytes<T, Result>(
+@_alwaysEmitIntoClient
+public func withUnsafeMutableBytes<T, E: Error, Result>(
+  of value: inout T,
+  _ body: (UnsafeMutableRawBufferPointer) throws(E) -> Result
+) throws(E) -> Result {
+  let pointer = UnsafeMutableRawPointer(Builtin.addressof(&value))
+  return try body(.init(start: pointer, count: MemoryLayout<T>.size))
```

Comment:
Any particular reason for writing this way instead of how the old thing was doing it? Not opposed, just curious 🙂

Reply:
The old way crashed the compiler; going straight to the builtin is a workaround.